diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+The loop runs 5 times. x = 4
```

Comment:
The loop only run's 3 times and x = 8. On each iteration of the loop, x = x + x is called. the value of x is initially 1 so x + x is like 1 + 1 and 'x =' is reassigning the value of 2 to x.